src/features/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
```

Comment:
While you did an amazing job working on the project requirements I noticed you  disabled eslint it is against microverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled eslint.

Reply:
I would implement the same here as well.